src/redux/saksoversikt/saksoversiktStateReducer.ts

```diff
@@ -0,0 +1,79 @@
+import { Dokument, DokumentMetadata } from '../../models/saksoversikt/dokumentmetadata';
```

Comment:
Looks good to me, men kanksje @jorundoa har noen kommentarer, han har jo gult belte i redux nå